ext_conf_template.txt

```diff
@@ -19,6 +19,9 @@ oidcClientKey =
 # cat=basic//4; type=string; label=LLL:EXT:oidc/Resources/Private/Language/locallang_db.xlf:settings.oidcClientSecret
 oidcClientSecret =
 
+# cat=basic//5; type=string; label=LLL:EXT:oidc/Resources/Private/Language/locallang_db.xlf:settings.oidcClientScopes
```

Comment:
Next time it would be best to combine the change here and in locallang_db.xlf since they belong naturally together.